arviz/plots/distcomparisonplot.py

```diff
@@ -26,42 +26,45 @@ def plot_dist_comparison(
-    than another), and plotted together, so three plots per distribution
+    than another), and plotted together, displaying a grid of three plots per
+    distribution.
 
     Parameters
     ----------
     data : InferenceData
-        :class:`arviz.InferenceData` object containing the posterior/prior data.
-    kind : str
-        kind of plot to display {"latent", "observed"}, defaults to 'latent'.
-        "latent" includes {"prior", "posterior"} and "observed" includes
-        {"observed_data", "prior_predictive", "posterior_predictive"}
+        Any object that can be converted to an :class:`arviz.InferenceData` object
+        containing the posterior/prior data. Refer to documentation of
+        :func:`arviz.convert_to_dataset` for details.
+    kind : {"latent", "observed"}, default "latent"
+        kind of plot to display The "latent" option includes {"prior", "posterior"},
+        and the "observed" option includes
+        {"observed_data", "prior_predictive", "posterior_predictive"}.
     figsize : (float, float), optional
-        Figure size. If None it will be defined automatically.
+        Figure size. If `None` it will be defined automatically.
     textsize : float
-        Text size scaling factor for labels, titles and lines. If None it will be
-        autoscaled based on ``figsize``.
-    var_names : str, list, list of lists
-        if str, plot the variable. if list, plot all the variables in list
-        of all groups. if list of lists, plot the vars of groups in respective lists.
+        Text size scaling factor for labels, titles and lines. If `None` it will be
+        autoscaled based on `figsize`.
+    var_names : list of str, optional
+        List of variables to plot.  If multiple datasets are supplied and `var_names` is not None,
+        will print the same set of variables for each dataset.  Defaults to `None`, which results in
+        all the variables being plotted.
     coords : dict
         Dictionary mapping dimensions to selected coordinates to be plotted.
         Dimensions without a mapping specified will include all coordinates for
         that dimension.
     combine_dims : set_like of str, optional
         List of dimensions to reduce. Defaults to reducing only the "chain" and "draw" dimensions.
-        See the :ref:`this section <common_combine_dims>` for usage examples.
-
+        See :ref:`this section <common_combine_dims>` for usage examples.
     transform : callable
-        Function to transform data (defaults to None i.e. the identity function)
+        Function to transform data (defaults to `None` i.e. the identity function).
     legend : bool
         Add legend to figure. By default True.
-    Labeller : Labeller, optional
-        Class providing the method ``make_pp_label`` to generate the labels in the plot.
+    labeller : Labeller, optional
+        Class providing the method ``make_label_vert`` to generate the labels in the plot titles.
```

Comment:
```suggestion
        Class providing the method ``make_pp_label`` to generate the labels in the plot titles.
```

labellers provide multiple methods to combine variable names, model names, coords and dims into the label that is eventually used. The method that is used depends on the function. Here, due to the possibility of different names, the pp one is used